File: src/main/java/com/chrisrm/idea/themes/literature/club/MTNatsukiTheme.java
package com.chrisrm.idea.themes.literature.club;

public final class MTNatsukiTheme extends MTDokiDokiTheme {
  public MTNatsukiTheme() {
    super("mt.natsuki", "Material Natsuki", false, "Natsuki");
  }

  @Override
  public String getBackgroundColorString() {
    return "fff3fc";
  }

  @Override
  public String getSecondaryBackgroundColorString() {
    return "ffceeb";
  }

  @Override
  public String getSecondaryForegroundColorString() {
    return "b9198d";
  }

  @Override
  public String getSelectionForegroundColorString() {
    return "fa6fe2";
  }

  @Override
  public String getSelectionBackgroundColorString() {
    return "ffd5f5";
  }

  @Override
  public String getTreeSelectionBackgroundColorString() {
    return "ff6eec";
  }

  @Override
  public String getMenuBarSelectionForegroundColorString() {
    return "ffffff";
  }

  @Override
  public String getMenuBarSelectionBackgroundColorString() {
    return "d9031a";
  }

  @Override
  public String getNotificationsColorString() {
    return "ffc7ec";
  }

  @Override
  public String getContrastColorString() {
    return "fdceff";
  }

  @Override
  public String getEditorTabColorString() {
    return getContrastColorString();
  }
}

File: src/main/java/com/chrisrm/idea/themes/literature/club/MTNatsukiTheme.kt
package com.chrisrm.idea.themes.literature.club

class MTNatsukiTheme : MTDokiDokiTheme("mt.natsuki", "Material Natsuki", false, "Natsuki") {

    override fun getBackgroundColorString(): String = "fff3fc"

    override fun getClubMember(): String = "natsuki.png"

    override fun getSecondaryBackgroundColorString(): String = "ffceeb"

    override fun getSecondaryForegroundColorString(): String = "b9198d"

    override fun getSelectionForegroundColorString(): String = "fa6fe2"

    override fun getSelectionBackgroundColorString(): String = "ffd5f5"

    override fun getTreeSelectionBackgroundColorString(): String = "ff6eec"

    override fun getMenuBarSelectionForegroundColorString(): String = "ffffff"

    override fun getMenuBarSelectionBackgroundColorString(): String = "d9031a"

    override fun getNotificationsColorString(): String = "ffc7ec"

    override fun getContrastColorString(): String = "fdceff"

    override fun getEditorTabColorString(): String = contrastColorString
}
